grpc/src/main/java/com/linecorp/armeria/server/grpc/NoopStatsContextFactory.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import com.google.instrumentation.stats.MeasurementMap;
+import com.google.instrumentation.stats.StatsContext;
+import com.google.instrumentation.stats.StatsContext.Builder;
+import com.google.instrumentation.stats.StatsContextFactory;
+import com.google.instrumentation.stats.TagKey;
+import com.google.instrumentation.stats.TagValue;
+
+/**
+ * A {@link StatsContextFactory} that returns a {@link StatsContext} that does nothing.
+ * Stats recording is handled within armeria itself.
+ */
+class NoopStatsContextFactory extends StatsContextFactory {
```

Comment:
Thought I had looked for this but obviously not well enough (even though it's even the same name by coincidence...), thanks for finding!